charts/rancher-monitoring/100.2.0+up40.1.2/app-README.md

```diff
@@ -14,6 +14,19 @@ The chart installs the following components:
 
 For more information, review the Helm README of this chart.
 
+## Upgrading from 100.1.0+up19.0.3 to 100.2.0+up40.1.2
+
+### Rebased the chart to upstream version 40.1.2. 
+### Noticeable changes:
+
+Prometheus-Operator:
+- Introduced a change to avoid name collisions, preventing AlertManagerConfig objects from generating identical references in the Alertmanager configuration. For example, take the follwing AlertManagerConfigs: 
+    * AlertmanagerConfig `foo-bar` in namespace `ns` defines a receiver with name `fred`.
+    * AlertmanagerConfig `bar` in namespace `ns-foo` defines a receiver with name `fred`.
+Without this patch, the resulting Alertmanagerconfiguration would have two receivers named `ns-foo-bar-fred`, which is invalid.
+With this patch, the receivers within alerting drivers would be named `ns/foo-bar/fred` and `ns-foo/bar/fred`, because `/` is not a valid character for namespaces. 
```

Comment:
Alerting Drivers should be capitalized, and IMO this whole text block should be re-written to emphasize 1) who needs to read this message (users of Alerting Drivers who specifically use the AlertmanagerConfig CR) 2) what behavioral change they need to make (will need to modify their Alertmanager configuration on upgrade) 2) why they need to do that (due to an upstream change), in that order.

We can then have a small blurb describing the upstream change (like the example here about how the names change).

But right now, this release note would be confusing for a user since it doesn’t tell them immediately if they are impacted and how to resolve it on or before upgrading without a technical understanding of Prometheus Operator.